src/UglyToad.PdfPig/Geometry/PdfPath.cs

```diff
@@ -5,27 +5,29 @@ namespace UglyToad.PdfPig.Geometry
     using System.Collections.Generic;
     using System.Linq;
     using System.Text;
+    using UglyToad.PdfPig.Core;
 
     /// <summary>
     /// A path in a PDF document, used by glyphs and page content.
     /// </summary>
     public class PdfPath
     {
-        private readonly List<IPathCommand> commands = new List<IPathCommand>();
+        public readonly List<IPathCommand> Commands = new List<IPathCommand>();
```

Comment:
I'd keep this `private readonly` and have a property that exposes a public immutable version:
```public IReadOnlyList<IPathCommand> Commands => commands;```